sites/mosul/configs/openmrs/initializer_config/ampathformstranslations/F71-POCUS_translations_en.json

```diff
@@ -4,121 +4,123 @@
+    "LV size": "LV size",
+    "Large": "Large",
+    "Left": "Left",
+    "Left - pleural effusion amount": "Left - pleural effusion amount",
+    "Lung": "Lung",
+    "Lung Ultrasound": "Lung Ultrasound",
+    "Lung sliding absent": "Lung sliding absent",
+    "Lung sliding absent site": "Lung sliding absent site",
+    "Mitral valve": "Mitral valve",
+    "Moderate": "Moderate",
+    "Moderate-Large": "Moderate-Large",
+    "Negative": "Negative",
+    "No": "No",
+    "No abnormality seen ": "No abnormality seen ",
+    "None of the above": "None of the above",
+    "Normal": "Normal",
+    "Other": "Other",
+    "POCUS": "POCUS",
+    "POCUS exams": "POCUS exams",
+    "POCUS impact on diagnosis/management": "POCUS impact on diagnosis/management",
+    "POCUS user": "POCUS user",
+    "Patient": "Patient",
+    "Pelvic Ultrasound": "Pelvic Ultrasound",
+    "Pelvic ultrasound findings": "Pelvic ultrasound findings",
+    "Pelvis": "Pelvis",
+    "Pericardial effusion": "Pericardial effusion",
+    "Pericardial effusion in cardiac USG": "Pericardial effusion in cardiac USG",
+    "Periportal/para-aortic lymph nodes": "Periportal/para-aortic lymph nodes",
+    "Pleural effusion": "Pleural effusion",
+    "Pleural effusion site": "Pleural effusion site",
+    "Pneumothorax excluded": "Pneumothorax excluded",
+    "Pneumothorax excluded (seashore sign)": "Pneumothorax excluded (seashore sign)",
+    "Pneumothorax suspected": "Pneumothorax suspected",
+    "Position of IUD": "Position of IUD",
+    "Positive": "Positive",
+    "RAN\u00a0": "RAN\u00a0",
+    "RAX": "RAX",
+    "RPO": "RPO",
+    "RUQ": "RUQ",
+    "RV size": "RV size",
+    "Reason": "Reason",
+    "Recommendations": "Recommendations",
+    "Reduced": "Reduced",
+    "Reduction / exclusion of differential diagnoses": "Reduction / exclusion of differential diagnoses",
+    "Reflux": "Reflux",
+    "Right": "Right",
+    "Right - pleural effusion amount": "Right - pleural effusion amount",
+    "Seashore sign": "Seashore sign",
+    "Small": "Small",
+    "Small consolidations": "Small consolidations",
+    "Small consolidations / nodules <0.5 cm": "Small consolidations / nodules <0.5 cm",
+    "Soft tissue": "Soft tissue",
+    "Summary of results": "Summary of results",
+    "Suspected abnormality of the uterus / annexes": "Suspected abnormality of the uterus / annexes",
+    "Suspected chronic lung pathology\u00a0": "Suspected chronic lung pathology\u00a0",
+    "Suspected effusions / ascites\u00a0": "Suspected effusions / ascites\u00a0",
+    "Suspected fracture / osteomyelitis": "Suspected fracture / osteomyelitis",
+    "Suspected heart disease": "Suspected heart disease",
+    "Telemedicine consultation will be done": "Telemedicine consultation will be done",
+    "Thickening": "Thickening",
+    "Trauma (abdominal/ thoracic)\u00a0": "Trauma (abdominal/ thoracic)\u00a0",
+    "Unclear ": "Unclear ",
+    "Yes": "Yes",
+    "nan": "nan"
```

Comment:
I missed this during the first look. Please replace occurrences of `nan` with `None`

Reply:
I am also adding update for Snakebite translation file too, and please note, the change was already made in the Snakebite form